src/ROUTES.js

```diff
@@ -78,17 +78,29 @@ export default {
     LOGIN_WITH_VALIDATE_CODE_2FA_WORKSPACE_CARD: 'v/:accountID/:validateCode/2fa/workspace/:policyID/card',
     ENABLE_PAYMENTS: 'enable-payments',
     WORKSPACE: 'workspace',
-    WORKSPACE_CARD: ':policyID/card',
-    WORKSPACE_PEOPLE: ':policyID/people',
     WORKSPACE_NEW: 'workspace/new',
-    getWorkspaceCardRoute: policyID => `workspace/${policyID}/card`,
-    getWorkspacePeopleRoute: policyID => `workspace/${policyID}/people`,
-    getWorkspaceInviteRoute: policyID => `workspace/${policyID}/invite`,
+    WORKSPACE_INITIAL: 'workspace/:policyID',
     WORKSPACE_INVITE: 'workspace/:policyID/invite',
+    WORKSPACE_SETTINGS: 'workspace/:policyID/settings',
+    WORKSPACE_CARD: 'workspace/:policyID/card',
+    WORKSPACE_REIMBURSE: 'workspace/:policyID/reimburse',
+    WORKSPACE_BILLS: 'workspace/:policyID/bills',
+    WORKSPACE_INVOICES: 'workspace/:policyID/invoices',
+    WORKSPACE_TRAVEL: 'workspace/:policyID/travel',
+    WORKSPACE_MEMBERS: 'workspace/:policyID/members',
+    WORKSPACE_BANKACCOUNT: 'workspace/:policyID/bankAccount',
+    getWorkspaceInitialRoute: policyID => `workspace/${policyID}`,
+    getWorkspaceInviteRoute: policyID => `workspace/${policyID}/invite`,
+    getWorkspaceSettingsRoute: policyID => `workspace/${policyID}/settings`,
+    getWorkspaceCardRoute: policyID => `workspace/${policyID}/card`,
+    getWorkspaceReimburseRoute: policyID => `workspace/${policyID}/reimburse`,
+    getWorkspaceBillsRoute: policyID => `workspace/${policyID}/bills`,
+    getWorkspaceInvoicesRoute: policyID => `workspace/${policyID}/invoices`,
+    getWorkspaceTravelRoute: policyID => `workspace/${policyID}/travel`,
+    getWorkspaceMembersRoute: policyID => `workspace/${policyID}/members`,
+    getWorkspaceBankAccountRoute: policyID => `workspace/${policyID}/bankAccount`,
```

Comment:
Hm, I see your point, but I think at that stage, you're pulling enough information out of the router (ie. the name of the page) that it almost doesn't make sense to keep hardly anything inside the router anymore. I'd like to keep it the way it is because I think it keeps most of the path defined inside the route file.